test/Sdk.Analyzers.Tests/BindingTypeNotSupportedTests.cs

```diff
@@ -7,332 +7,153 @@
-    //                     public void Run([TestTrigger()] string message)
-    //                     {
-    //                     }
-    //                 }
-    //             }";
-
-    //         var test = new AnalyzerTest
-    //         {
-    //             ReferenceAssemblies = ReferenceAssemblies.Net.Net50.WithPackages(ImmutableArray.Create(
-    //                                     new PackageIdentity("Microsoft.Azure.Functions.Worker", "1.15.0-preview1"))),
-    //             TestCode = testCode
-    //         };
-
-    //         // test.ExpectedDiagnostics is an empty collection.
-
-    //         await test.RunAsync();
-    //     }
-
-    //     [Fact]
-    //     public async Task TestAttribute_InvalidBindingType_DiagnosticsExpected()
-    //     {
-    //         string testCode = @"
-    //             using System;
-    //             using Microsoft.Azure.Functions.Worker;
-    //             using Microsoft.Azure.Functions.Worker.Core;
-    //             using Microsoft.Azure.Functions.Worker.Converters;
-    //             using Microsoft.Azure.Functions.Worker.Extensions.Abstractions;
-
-    //             namespace Microsoft.Azure.Functions.Worker
-    //             {
-    //                 [AllowConverterFallback(false)]
-    //                 [InputConverter(typeof(TestConverter))]
-    //                 public class TestTriggerAttribute : TriggerBindingAttribute
-    //                 {
-    //                 }
-
-    //                 [SupportsDeferredBinding]
-    //                 [SupportedConverterType(typeof(string))]
-    //                 [SupportedConverterType(typeof(bool))]
-    //                 public class TestConverter
-    //                 {
-    //                 }
-    //             }
-
-    //             namespace FunctionApp
-    //             {
-    //                 public class SomeFunction
-    //                 {
-    //                     [Function(nameof(SomeFunction))]
-    //                     public void Run([TestTrigger()] BinaryData message)
-    //                     {
-    //                     }
-    //                 }
-    //             }";
-
-    //         var test = new AnalyzerTest
-    //         {
-    //             ReferenceAssemblies = ReferenceAssemblies.Net.Net50.WithPackages(ImmutableArray.Create(
-    //                                     new PackageIdentity("Microsoft.Azure.Functions.Worker", "1.15.0-preview1"))),
-    //             TestCode = testCode
-    //         };
-
-    //         test.ExpectedDiagnostics.Add(Verify.Diagnostic().WithSeverity(Microsoft.CodeAnalysis.DiagnosticSeverity.Warning)
-    //             .WithSpan(29, 68, 29, 75).WithArguments("BinaryData", "TestTriggerAttribute"));
-
-    //         await test.RunAsync();
-    //     }
-
-    //     /*  The following tests use real binding attributes from extensions.
-    //         We don't currently have a real binding that has SupportedConverterType
```

Comment:
Deleting these extra tests as the cases we wanted to test are already covered by the above and this is not providing any value